example/config/README.md

```diff
@@ -0,0 +1,249 @@
+
+
+### Model Scoring
+How each trained model is scored?
+
+Each entry in `testing_metric_groups` needs a list of one of the metrics defined in catwalk.evaluation.ModelEvaluator.available_metrics (contributions welcome!) Depending on the metric, either thresholds or parameters.
+
+`Parameters`: specify any hyperparameters needed. For most metrics, which are simply wrappers of sklearn functions, these are passed directly to sklearn. 
+
+- `thresholds` are more specific: The list is dichotomized and only the top percentile or top n entities are scored as positive labels
+
+subsets, if passed, will add evaluations for subset(s) of the predictions to the subset_evaluations tables, using the same testing and training metric groups as used for overall evaluations but with any thresholds reapplied only to entities in the subset on the relevant as_of_dates. For example, when calculating **precision@5_pct** for the subset of women, the ModelEvaluator will count as positively labeled the top 5% of women, rather than any women in the top 5% overall. This is useful if, for example, different interventions will be applied to different subsets of entities (e.g., one program will provide subsidies to the top 500 women with children and another program will provide shelter to the top 150 women without children) and you would like to see whether a single model can be used for both applications. Subsets can also be used to see how a model's performance would be affected if the requirements for intervention eligibility became more restricted.
+
+### Individual Importances
+How feature importances for individuals should be computed. This entire section can be left blank, in which case the defaults will be used.
+
+- `individual_importance`:
+    - `methods`: Refer to *how to compute* individual importances. Each entry in this list should represent a different method. Available methods are in the catwalk library's: `catwalk.individual_importance.CALCULATE_STRATEGIES` list. Will default to `uniform`, or just the global importances. Empty list means don't calculate individual importances.
+    - `n_ranks`: The number of top features per individual to compute importances for. Will default to 5.
+
+
+## Audition Configuration
+Also check out the example file `audition.yaml`.
+
+### Choose Model Groups
+Audtion needs a buch of `model_group_id's to help users select the models.
+
+- `model_groups`:
+    - `query`: The query is to choose what the model groups you want to include in the first round of model selection.
+
+
+### Choose Timestamps/Train end times
+The timestamps when audition happens for each model group.
+
+- `time_stamps`:
+    - `query`: There's a hard rule in Audition that all of the chosen model groups for audition should have the same train end times as the timestamps or the subset of the timestamps from this query, otherwise those model groups with unmatched train end times will be pruned in the first round.
```

Comment:
I would say model groups with _missing_ timestamps will be pruned (it's ok to have extra, right?)